functions/Export-DbaInstance.ps1

```diff
@@ -163,6 +163,10 @@ function Export-DbaInstance {
         $ScriptingOptions = New-Object Microsoft.SqlServer.Management.Smo.ScriptingOptions
         $ScriptingOptions.ScriptBatchTerminator = $true
 
+        # The way this command is currently written it makes sense for it to always append, since the files are dynamically created and some of the exported objects are gathered in multiple steps and then written to the same file.
+        # This issue is discussed at: https://github.com/sqlcollaborative/dbatools/pull/6919 and https://github.com/sqlcollaborative/dbatools/issues/6918
+        $Append = $true
+        $ScriptingOptions.AppendToFile = $true
```

Comment:
We can't provide a parameter of `-Append` and always set it to true, that defeats the purpose of a switch parameter. As-is if this is going to be done the parameter has to be removed because it gives a false sense to users that it will not append by default.

Reply:
Going to move this discussion to the issue because this can't be done using this method.

Reply:
I agree here